ghost/email-service/lib/email-event-storage.js

```diff
@@ -61,7 +66,11 @@ class EmailEventStorage {
     }
 
     async handleComplained(event) {
-        return this.unsubscribeFromNewsletters(event);
```

Comment:
Do we want a feature flag check here?

Reply:
Discussed in slack: we're good to store the new data without a flag check, as it'll give us a backlog of events if release is delayed.